openwisp_ipam/api/views.py

```diff
@@ -220,19 +234,20 @@ def post(self, request, *args, **kwargs):
         return Response(None)
 
 
-class ImportSubnetView(CreateAPIView):
+class ImportSubnetView(ImportSubnetCSVMixin, CreateAPIView):
     subnet_model = Subnet
     queryset = Subnet.objects.none()
     serializer_class = ImportSubnetSerializer
     authentication_classes = (BearerAuthentication, SessionAuthentication)
     permission_classes = (DjangoModelPermissions,)
 
     def post(self, request, *args, **kwargs):
+        super().post(request)
```

Comment:
I'm not sure this is the best thing to do, are you calling this in order to trigger the permission and authorization checks?
There's a way to do that without executing the whole post method, which may trigger also other actions we do not need here.

Reply:
> are you calling this in order to trigger the permission and authorization checks?

Yes, I changed it now. Thanks